18F/identity-idp
# LG-7355 GettingStarted A/B analytics

## 🎫 Ticket

[LG-7355](https://cm-jira.usa.gov/browse/LG-7355)

## 🛠 Summary of changes

Add analytics for GettingStarted A/B Test.
Also change the test discriminator from session id to uuid as specified in the ticket.

Along the way:
- Add a more general mechanism for tracking A/B tests in IdV controller analytics.
  - Fold in acuant_sdk_ab_test_analytics_args
- Test new method ab_test_analytics_args separately and then mock in controller specs 
  - Note that analytics_spec will still need to be updated by hand when A/B test analytics change 
- Remove StepUtilitiesConcern and put methods in IdvSession concern. 
  - I've been meaning to do this for a while and I was editing all the controllers anyway...
- Remove unneeded native_camera_ab_testing_variables method from hybrid_mobile document capture controller
  - I've been eyeing that for a while too.

## 📜 Testing Plan

- [ ] Create account
- [ ] Complete full IdV
- [ ] Look at events.log, confirm that `acuant_sdk_upgrade_ab_test_bucket`  and `getting_started_ab_test_bucket` are included
